migrations/202010201032128-update-events-add-blockchain-id.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        await queryInterface.addColumn(
+            'events',
+            'blockchain_id',
+            {
+                type: Sequelize.STRING,
+            },
+        );
+
+        await queryInterface.sequelize.query('UPDATE events SET blockchain_id = \'rinkeby\'');
```

Comment:
Should we create a new table for blockchain details like frequency for fetching new events, endpoints and others, or is it unnecessary?